### ISSUE TITLE
Add: 9fit

### ISSUE THREAD

**Issue by n0127** [cdnjs/packages#1607]
### Library name

9fit

### Library license

https://github.com/n0127/fit/blob/main/LICENSE

### Library source

https://github.com/n0127/fit

### Files to include

source/*

**Comment by MattIPv4** [comment]
👋 The `source` directory does not exist in this package. Please could you clarify what files you are wanting to be hosted on cdnjs for this package?

**Comment by n0127** [comment]
> 👋 该`source`目录在该包中不存在。请您澄清一下您希望在 cdnjs 上托管此软件包的哪些文件？



> 👋 The `source` directory does not exist in this package. Please could you clarify what files you are wanting to be hosted on cdnjs for this package?
thanks i want to open

that 
souce:
/static/js/player.js
/static/js/playerconfig.js
/static/player/ness.js
/template/ness/chat/css/*.css

**Comment by n0127** [comment]
> https://www.npmjs.com/package/9fit

Because I am developing for them personally, the number of users has increased recently, but my ability to provide services has become weaker, and I did not buy a server, so I hope you can add my warehouse

**Comment by MattIPv4** [comment]
Would you be able to provide some more context on what this library is, and how end-users of the library will be using all of these files? (It seems like a lot of files to including on a website to use your library?)

**Comment by n0127** [comment]
> Would you be able to provide some more context on what this library is, and how end-users of the library will be using all of these files? (It seems like a lot of files to include on a website to use your library?)
Yes, I provide many complete solutions for webmasters of many small websites. You can provide me with the number of citations per week. Please help me add them. Thank you. Currently, the main focus is to add them so that CDNs within China can be referenced

**Comment by MattIPv4** [comment]
Hey! That didn't really help explain what this library is that you're asking us to host... what does this library do, what is its function? Why does the library need so many files to function?

Your response makes it sound like this isn't actually a library for people to use, but rather you just trying to use npm/cdnjs to just random files for your clients?

**Comment by n0127** [comment]
> Hey! That didn't really help explain what this library is that you're asking us to host... what does this library do, what is its function? Why does the library need so many files to function?
> 
> Your response makes it sound like this isn't actually a library for people to use, but rather you just trying to use npm/cdnjs to just random files for your clients?

This library is a player content that helps webmasters play, which reduces their repetitive development and iteration. I use my spare time to help them complete it, so they need domestic CDN methods. Because China does not have very good support for some unregistered networks overseas, many webmasters want to use my library, and loading resources is either very slow or 404, So I hope CDNJS can help me include this library, and there are also examples in China, such as Bytes http://cdn.bytedance.com/ Clicking 'add' here requires you to help me add a new library, which is why I hope to be included. CDN is provided by Byte (which is an overseas ticketock). Thank you
